src/scribe.js

```diff
@@ -133,7 +133,9 @@ define([
   };
 
   Scribe.prototype.setHTML = function (html, skipFormatters) {
-    this._lastItem.content = html;
+    if (this.options.undo.enabled) {
```

Comment:
I believe this works but since I'm not super well versed I'd be more tempted to do something closer to:

`this._lastItem.content = {content: ''}`

This works for me and I have it deployed to (well okay, Beta) production without any issues. Not sure if there are any other weird codepaths that we haven't found yet where _lastItem is depended on without a check somewhere.